src/test/kotlin/com/ekino/oss/gradle/plugin/quality/GradleVersionsCompatibilityTest.kt

```diff
@@ -14,7 +14,7 @@ class GradleVersionsCompatibilityTest {
     @TempDir
     lateinit var tempDir: File
 
-    @ValueSource(strings = ["5.3.1", "5.6.1"])
+    @ValueSource(strings = ["5.3.1", "5.6.1", "5.6.2"])
```

Comment:
Hi,
We don't actually need to test both `5.6.1` and `5.6.2` while there are very close.
You can remove `5.6.1` safely.

Reply:
Hi, 
Ok, thank you.